middlewares/jwtValidator.js

```diff
@@ -0,0 +1,24 @@
+const jwtUtil = require('../lib/utils/jwtUtil');
+
+const jwtValidator = (req, res, next) => {
+  // Get the token sent from anywhere(i.e header,body or query)
+  const token = req.body.token || req.query.token || req.headers.token || req.params;
+  if (token) {
+    // Decode token if it is not empty and call next function
+    try {
+      req.decodedToken = jwtUtil.verifyToken(token);
+      next();
+    } catch (err) {
+      // Return an error if it occurs
+      res.json({
```

Comment:
What is the status code being sent to the user here?

Reply:
I have included the response code